modules/nodes-cluster-worker-latency-profiles-using-at-creation.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * scaling-worker-latency-profiles.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nodes-cluster-worker-latency-profiles-using-at-creation_{context}"]
+= Implementing worker latency profiles at cluster creation
+
+[IMPORTANT]
+====
+To edit the configuration of the installer, you will first need to use the command `openshift-install create manifests` to create the default node manifest as well as other manifest YAML files. This file structure must exist before we can add workerLatencyProfile. The platform on which you are installing may have varying requirements. Refer to the Installing section of the documentation for your specific platform.
+====
+
+The `workerLatencyProfile` must be added to the manifest in the following sequence:
+
+. Create the manifest needed to build the cluster.
+. You will use the file `<cluster-install-dir>/manifests/config-node-default-profile.yaml` to add the name of the profile.
+. Edit the `node.config` object in this file to add the name of the profile. You can change the profile at any time as latency increases or decreases.
```

Comment:
this node.config object doesnt exist at this sep, the line above is instead of this step entirely

Reply:
OK!

Reply:
Thank you @paigerube14 this is done.